core/src/main/clojure/xtdb/vector.clj

```diff
@@ -268,11 +268,10 @@
         (writeObject [_ v]
           (let [^PeriodDuration period-duration v
```

Comment:
could you validate that PeriodDuration doesn't have a year/month part here, too? these won't hit your `IntervalDayTime` check.

Reply:
I am just wondering how would we hit this?